chef_master/source/deprecations_locale_lc_all.rst

```diff
@@ -0,0 +1,42 @@
+=======================================================================
+Deprecation: Deprecation of lc_all from locale resource (CHEF-27)
+=======================================================================
+`[edit on GitHub] <https://github.com/chef/chef-web-docs/blob/master/chef_master/source/deprecations_locale_lc_all.rst>`__
+
+.. tag deprecations_locale_lc_all
+
+Setting the ``LC_ALL`` variable is NOT recommended. As a system-wide setting, ``LANG`` should provide the desired behavior. ``LC_ALL`` is intended to be used for temporarily troubleshooting issues rather than an everyday system setting.
+Changing ``LC_ALL`` can break Chef's parsing of command output in unexpected ways. Use one of the more specific ``LC_`` properties as needed.
+This deprecation warning was added in Chef 14.12. Support for property ``lc_all`` will be removed for Chef 16.0.
+
+.. end_tag
+
+
+Remediation
+=======================================================================
+
+Setting *LC_* variables varies by platform, but common locations to configure *LC_* variables are:
+
+.. code-block:: ruby
```

Comment:
@Nimesh-Msys What does this code block do? For documentation purposes, a sentence should be added to describe what the code block does. Thank you in advance!

Reply:
Thanks for pointing it out, 'tws left over there while refactor.
Actually, we wanted to convey this to users, that if they want to update LC_ALL, do it this way or else use file resource on those mentioned list of files.
It should be like:

```
Remediation
=======================================================================


Set `LC_ALL` in current shell as:

.. code-block:: bash
	export LC_ALL="<locale_name>"

To check the locale value, run

.. code-block:: bash
	locale -v

You can also use *file* resource and add this variable in any other file of your choice and then can source that file to reflect changes.

.. code-block:: bash
	file "<path_to_file>" do
	  content "LC_ALL=<locale_name>"
	end

Where `path_to_file` could be any one of:

1. /etc/default/locale
2. /etc/sysconfig/i18n
3. /etc/environment

Setting *LC_* variables varies by platform, but these are the common locations to configure *LC_* variables.

.. warning:: Be cautious while using *file* resource on any existing or above listed files. As it replaces all the file contents, you may loose any previous information.
```

Reply:
Added.